airbrus.mt

```diff
@@ -68,6 +68,55 @@ def UTF8JSON :DeepFrozen := composeCodec(UTF8, JSON)
 def makeLineTube() as DeepFrozen:
     return makePumpTube(makeSplitPump(b`$\n`))
 
+def make_giftExchange() :Any as DeepFrozen:
+    def givers  := [].asMap().diverge()
+    def wanters := [].asMap().diverge()
+        
+    def give(giver :Str, acceptor :Str, giftName :Str, gift :Any) :Void :
+        def giftBox := gift  # Allways wrap your presents! (For when gifts are promises)
```

Comment:
This does nothing. If you want to ensure that `gift` is near, guard it as `gift :Near`.